app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,34 @@
+class CommentsController < ApplicationController
+
+  def create
+    case params[:comment_type]
```

Comment:
You probably don't need to use params[:comment_type] - if there's a params[:answer_id] present then this is a comment on an answer.